client/js/components/admin/instances/new_domain_modal.jsx

```diff
@@ -15,10 +16,11 @@ export class NewDomainModal extends React.Component {
     })
   };
 
-  renderSettings(){
-    return _.map(this.props.settings, (value, key) => {
-      return (<input key={key} type="hidden" value={value} name={key} />);
-    });
+  renderSettings() {
+    return _.map(
+      this.props.settings,
+      (value, key) => <input key={key} type="hidden" value={value} name={key} />
```

Comment:
Refactored to return a single expression. Complies with http://eslint.org/docs/rules/arrow-body-style.